_api_app/app/Sites/Sections/Entries/SectionEntriesDataService.php

```diff
@@ -4,11 +4,157 @@
+                                            ]
+                                        ]
+                                    ]
+                                ],
+                                '@attributes' => [
+                                    'type' => 'object',
+                                    'properties' => [
+                                        'autoplay' => ['type' => 'integer'],
+                                        'fullscreen' => ['type' => 'string', 'enum' => ['yes', 'no']],
+                                        'link_address' => ['type' => 'string'],
+                                        'linkTarget' => ['type' => 'string'],
+                                        'row_gallery_padding' => ['type' => 'string', 'format' => 'css-unit'],
+                                        'size' => ['type' => 'string'],
+                                        'slide_numbers_visible' => ['type' => 'string', 'enum' => ['yes', 'no']],
+                                        'type' => ['type' => 'string'],  /** @todo: figure out what types can there be */
+                                    ]
+                                ]
+                            ]
+                        ],
+                        'content' => [
+                            'type' => 'object',
+                            'properties' => [
+                                'description' => ['type' => 'string'],
+                                'fixed' => ['type' => 'integer'],
+                                'positionXY' => ['type' => 'string', 'pattern' => '^[0-9]+,[0-9]+$'],
+                                'title' => ['type' => 'string'],
+                                'url' => ['type' => 'string', 'format' => 'URI'],
+                                'width' => ['type' => 'string', 'format' => 'css-unit']
+                            ]
+                        ],
+                        'updated' => ['type' => 'string', 'format' => 'berta-date-time'],
+                        'tags' => [
+                            'type' => 'object',
+                            'properties' => [
+                                'tag' => ['type' => 'array', 'items' => ['type' => 'string']]
+                            ]
+                        ],
+                        'marked' => ['type' => 'integer']
+                    ]
+                ]
+            ],
+            '@attributes' => [
+                'type' => 'object',
+                'properties' => [
+                    'section' => [
+                        'type' => 'string',
+                        '$comment' => 'name of section these entries belong to. See $this->SECTION_NAME'
+                    ],
+                    'version' => [
+                        'type' => 'string',
+                        '$comment' => 'Version of berta, this was created in'
+                    ],
+                    'last_upd_ver' => ['type' => 'string']
```

Comment:
@uldisrudzitis could you add more info on what attributes on the  `<entry>` tag mean? Like this `last_upd_ver`?